docs/extending/parcellations.rst

```diff
@@ -52,13 +53,13 @@ first label in this list corresponds to the first integer label in the
 parcellation and so on).
 
 Lastly, we specify the ``space`` that the parcellation is in, for example,
-``"MNI"`` or ``"Native"`` (scanner-native space).
+``"MNI152NLin2009cAsym"`` or ``"native"`` (scanner-native space).
 
 Step 1: Prepare code to register a parcellation
 -----------------------------------------------
 
-Now we know everything that we need to know to make sure junifer can use our
-own parcellation to compute any parcellation-based marker. For example,
+Now we know everything that we need to know to make sure ``junifer`` can use our
+own parcellation to compute any parcellation-based Marker. For example,
 a simple example could look like this:
```

Comment:
"For example, a simple example could look like this:" -> "A simple example could look like this:"